wpilibj/src/test/java/edu/wpi/first/wpilibj/MockSpeedController.java

```diff
@@ -8,12 +8,12 @@
 package edu.wpi.first.wpilibj;
 
 public class MockSpeedController implements SpeedController {
+  private double m_speed = 0.0;
+  private boolean m_isInverted = false;
```

Comment:
How is it redundant? They should be initialized before use, and this is the only place in MockSpeedController that happens.